flow_logs.tf

```diff
@@ -0,0 +1,64 @@
+resource "aws_flow_log" "flow_logs" {
+  count = var.enable_flow_logs ? 1 : 0
+
+  iam_role_arn    = aws_iam_role.flow_logs.arn
+  log_destination = aws_cloudwatch_log_group.flow_logs.arn
```

Comment:
I wonder if the log destination should be configurable? I was wondering if we wanted ours to go to s3 instead